src/java/org/apache/cassandra/utils/FBUtilities.java

```diff
@@ -136,6 +137,8 @@ public static int getAvailableProcessors()
 
     public static final int MAX_UNSIGNED_SHORT = 0xFFFF;
 
+    public static final int ASM_BYTECODE_VERSION = Opcodes.ASM9;
```

Comment:
Again, this is copied from `main-5.0`, but I would argue that it might be better to keep it at `ASM7`, since ASM 7 fully supports JDK 11 (source: [ASM changelog](https://asm.ow2.io/versions.html)).

As far as I understand, this "API version" opcode gets passed to the visitors to ensure that we won't forget to override methods as we add support for newer JDK versions. For example, we currently do not override `ClassVisitor#visitPermittedSubclass()`; this is fine as long as we are parsing JDK 11 bytecode, because sealed class are a JDK 16+ feature. But if we upgrade the UDF language later, we might want to override it or we'll miss parts of the bytecode. I feel that by upgrading the API version early, we increase the chance that we might forget.